101-route-server-quagga/azuredeploy.json

```diff
@@ -0,0 +1,467 @@
+                "description": "address prefix assigned to the subnet2"
+            }
+        },
+        "subnet3Name": {
+            "type": "string",
+            "defaultValue": "subnet3",
+            "metadata": {
+                "description": "name of the subnet3"
+            }
+        },
+        "subnet3Prefix": {
+            "type": "string",
+            "defaultValue": "10.1.3.0/24",
+            "metadata": {
+                "description": "address prefix assigned to the subnet3"
+            }
+        },
+        "subnet4Name": {
+            "type": "string",
+            "defaultValue": "subnet4",
+            "metadata": {
+                "description": "name of the subnet4"
+            }
+        },
+        "subnet4Prefix": {
+            "type": "string",
+            "defaultValue": "10.1.4.0/24",
+            "metadata": {
+                "description": "address prefix assigned to the subnet4"
+            }
+        },
+        "gatewaysubnetPrefix": {
+            "type": "string",
+            "defaultValue": "10.1.5.0/24",
+            "metadata": {
+                "description": "address prefix assigned to the GatewaySubnet"
+            }
+        },
+        "routeServerName": {
+            "type": "string",
+            "defaultValue": "routesrv1",
+            "metadata": {
+                "description": "Route Server name"
+            }
+        },
+        "peering1IP": {
+            "type": "string",
+            "defaultValue": "10.1.4.10",
+            "metadata": {
+                "description": "private IP address of the VM running Quagga. it is the BGP remote peer of Route Server. The value is referenced in the quaggadeploy.sh script"
+            }
+        },
+        "peering1ASN": {
+            "type": "int",
+            "defaultValue": 65001,
+            "metadata": {
+                "description": "ASN assigned to Quagga. The value is referenced in the script quaggadeploy.sh"
+            }
+        },
+        "rsBGPconnectionName": {
+            "type": "string",
+            "defaultValue": "bgp-conn1",
+            "metadata": {
+                "description": "Route Server BGP connection name to establish the peer with Quagga"
+            }
+        },
+        "vHubIPconfigName": {
+            "type": "string",
+            "defaultValue": "ipconfig1",
+            "metadata": {
+                "description": "Route Server configuration name"
+            }
+        },
+        "vmName": {
+            "type": "string",
+            "defaultValue": "vm1",
+            "metadata": {
+                "description": "Virtual Machine name"
+            }
+        },
+        "vmPrivIP": {
+            "type": "string",
+            "defaultValue": "10.1.4.10",
+            "metadata": {
+                "description": "private IP of the VM. This IP has to match with the IP specified in the quaggadeploy.sh bash script"
+            }
+        },
+        "publisher": {
```

Comment:
set allowedValues on the vm image properties

Reply:
added the variable:
"imageReference": {
            "publisher": "Canonical",
            "offer": "0001-com-ubuntu-server-focal",
            "sku": "20_04-lts",
            "version": "latest"
        },